tv/src/main/java/butter/droid/tv/presenters/MediaDetailsDescriptionPresenter.java

```diff
@@ -30,18 +30,6 @@ protected void onBindDescription(ViewHolder viewHolder, Object item) {
         if (itemWrapper != null) {
             Media media = itemWrapper.getMedia();
             viewHolder.getTitle().setText(media.getTitle());
-
-            Genre[] genres = media.getGenres();
```

Comment:
`genresText` at the end was not used. what was the purpose of it. should we add the genres in the body? for now I just removed the code. happy to put it back in the body of the viewholder

Reply:
That is definitely something we want to show to users, but I'm not yet sure where and how. That goes for the whole app not just this screen :). For now it is ok to be removed